compose/snippets/src/main/java/com/example/compose/snippets/layouts/CommonLayoutExamples.kt

```diff
@@ -149,4 +156,24 @@ fun Layout_StretchMiddleItem() {
         Spacer(modifier = itemModifier.background(MaterialColors.Pink200))
     }
     // [END android_compose_layout_stretch_middle]
+
+    // [START android_compose_layout_scrollable_grid]
+    val itemsList = (0..15).toList()
+
+    val itemModifier = Modifier.border(1.dp, Color.Blue).width(80.dp).wrapContentSize()
+
+    LazyHorizontalGrid(
+        rows = GridCells.Fixed(3),
+        horizontalArrangement = Arrangement.spacedBy(16.dp),
+        verticalArrangement = Arrangement.spacedBy(16.dp)
+    ) {
+        items(itemsList) {
+            Text("Item is $it", itemModifier)
+        }
+
+        item {
+            Text("Single item", itemModifier)
+        }
+    }
+    // [END android_compose_layout_scrollable_grid]
```

Comment:
This should be a new composable and not part of common layout existing composable snippet. It should be part of the `Lazy Layout` snippets.

Reply:
Thanks. Will fix.

Reply:
Where are the Lazy Layout snippets? There doesn't seem to be such a file in the layouts/ directory.

Reply:
Think I found the file you mean: .../lists/LazyListSnippets.kt. Will move it to there.

Reply:
Its moved but still exists in this file, be sure to delete it from here too.